datahub-web/@datahub/data-models/LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
```

Comment:
Is this our license? If so it should Apache instead

Reply:
Haha, default ember I guess. Changed.